crates/sui-indexer-alt-jsonrpc/src/config.rs

```diff
@@ -279,10 +283,29 @@ impl CoinsLayer {
     }
 }
 
+impl NodeConfig {
+    pub fn create_fullnode_rpc_client(
```

Comment:
nit (optional): just `client`? `create_`/`make_`/`new_` prefixes are not idiomatic Rust IIRC, and the rest is implied by the fact that the receiver of this function is a `NodeConfig`.

Doc comment wouldn't go amiss though.